source/clear-linux/guides/maintenance/download-verify-uncompress-windows.rst

```diff
@@ -0,0 +1,66 @@
+.. _download-verify-uncompress-windows:
+
+Download, verify, and uncompress a Clear Linux image on Windows
+###############################################################
+
+This section explains the types of |CLOSIA| images available, where to download
+them, how to verify the integrity of an image, and how to uncompress it.
+
+Alternative instructions for other operating systems are available:
+
+* :ref:`download-verify-uncompress-linux`
+* :ref:`download-verify-uncompress-mac`
+
+.. include:: types-of-cl-images.rst
+  :start-after: types-of-cl-images
+
+Download the latest Clear Linux image
+=====================================
+
+Go to the Clear Linux `image`_ repository and download the desired type of image.
+
+.. _verify-image-checksum-on-windows:
+
+Verify the integrity of the Clear Linux image
+=============================================
+
+Before you use a downloaded |CL| image, it is recommended that you verify its 
+integrity to eliminate the chance, albeit small, that it might be corrupted due 
```

Comment:
Keep language simple, concise, and clear:
integrity. Thus, you eliminate the small chance of a corrupted image due to download issues.